### ISSUE TITLE
crash report

### ISSUE THREAD

**Issue by s3786488** [FiskFille/TransformersMod#160]
i open the game with transformers mod 0.5.10 in and then press play, it loads and then it gives me a crash report

**Comment by Gegy** [comment]
Can you please provide us with the crash report it has given you? :)

**Comment by s3786488** [comment]
---- Minecraft Crash Report ----
// Everything's going to plan. No, really, that was supposed to happen.

Time: 4/2/16 10:24 AM
Description: Initializing game

java.lang.RuntimeException: No more entity indicies left
    at cpw.mods.fml.common.registry.EntityRegistry.findGlobalUniqueEntityId(EntityRegistry.java:322)
    at fiskfille.tf.common.entity.TFEntities.registerEntity(TFEntities.java:34)
    at fiskfille.tf.common.entity.TFEntities.registerEntities(TFEntities.java:14)
    at fiskfille.tf.TransformersMod.preInit(TransformersMod.java:92)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
    at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
    at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
    at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:556)
    at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at cpw.mods.fml.common.registry.EntityRegistry.findGlobalUniqueEntityId(EntityRegistry.java:322)
    at fiskfille.tf.common.entity.TFEntities.registerEntity(TFEntities.java:34)
    at fiskfille.tf.common.entity.TFEntities.registerEntities(TFEntities.java:14)
    at fiskfille.tf.TransformersMod.preInit(TransformersMod.java:92)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
    at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
    at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
    at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:556)
    at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)

-- Initialization --
Details:
Stacktrace:
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.7.10
    Operating System: Windows 10 (x86) version 10.0
    Java Version: 1.8.0_25, Oracle Corporation
    Java VM Version: Java HotSpot(TM) Client VM (mixed mode), Oracle Corporation
    Memory: 92838808 bytes (88 MB) / 348135424 bytes (332 MB) up to 523501568 bytes (499 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx512M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1558 Optifine OptiFine_1.7.10_HD_U_D1 20 mods loaded, 20 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCH mcp{9.05} [Minecraft Coder Pack](minecraft.jar) 
    UCH FML{7.10.99.99} [Forge Mod Loader](forge-1.7.10-10.13.4.1558-1.7.10.jar) 
    UCH Forge{10.13.4.1558} [Minecraft Forge](forge-1.7.10-10.13.4.1558-1.7.10.jar) 
    UCH CodeChickenCore{1.0.7.47} [CodeChicken Core](minecraft.jar) 
    UCH NotEnoughItems{1.0.5.120} [Not Enough Items](NotEnoughItems-1.7.10-1.0.5.120-universal.jar) 
    UCH EE3{0.3.506} [Equivalent Exchange 3](EquivalentExchange3-1.7.10-0.3.506.jar) 
    UCH iChunUtil{4.2.2} [iChunUtil](iChunUtil-4.2.2.jar) 
    UCH InventoryPets{1.4.2} [Inventory Pets](inventorypets-1.7.10-1.4.2-universal.jar) 
    UCH IronChest{6.0.41.729} [Iron Chest](ironchest-1.7.10-6.0.41.729-universal.jar) 
    UCH journeymap{5.1.4} [JourneyMap](journeymap-1.7.10-5.1.4-unlimited.jar) 
    UCH legends{1.5} [Legends Mod](Legends Core-1.5 %281%29.jar) 
    UCH malisiscore{1.7.10-0.12.9} [MalisisCore](malisiscore-1.7.10-0.12.9.jar) 
    UCH malisisdoors{1.7.10-1.10} [Malisis' Doors](malisisdoors-1.7.10-1.10.jar) 
    UCH OreSpawn{1.7.10.20.3} [OreSpawn](orespawn-1.7.10-20.3.zip) 
    UCH pacman{1.0.8} [Killer Pacman](PacmanMod-1.0.8.jar) 
    UCH PortalGun{4.0.0-beta-6} [PortalGun](PortalGun-4.0.0-beta-6.jar) 
    UCH ironman{4.3.3} [Iron Man Armor Extension](Superheroes Unlimited Mod-1.7.10-4.3.5 %281%29.jar) 
    UCH sus{4.3.5} [Superheroes Unlimited Mod](Superheroes Unlimited Mod-1.7.10-4.3.5 %281%29.jar) 
    UCH susaddon{4.3.5} [Superheroes Unlimited Mod (Part 2)](Superheroes Unlimited Mod-1.7.10-4.3.5 %281%29.jar) 
    UCE transformers{${version}} [Transformers Mod](Transformers-1.7.10-0.5.10.jar) 
    GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13399 Compatibility Profile Context 15.201.1151.1008' Renderer: 'AMD Mobility Radeon HD 5000 Series'
    Launched Version: 1.7.10-Forge10.13.4.1558-1.7.10
    LWJGL: 2.9.1
    OpenGL: AMD Mobility Radeon HD 5000 Series GL version 4.5.13399 Compatibility Profile Context 15.201.1151.1008, ATI Technologies Inc.
    GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

```
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: []
Current Language: English (US)
Profiler Position: N/A (disabled)
Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
Anisotropic Filtering: Off (1)
OptiFine Version: OptiFine_1.7.10_HD_U_D1
Render Distance Chunks: 2
Mipmaps: 4
Anisotropic Filtering: 1
Antialiasing: 0
Multitexture: false
OpenGlVersion: 4.5.13399 Compatibility Profile Context 15.201.1151.1008
OpenGlRenderer: AMD Mobility Radeon HD 5000 Series
OpenGlVendor: ATI Technologies Inc.
CpuCount: 8
```

(if this is not a/the crash report plz let me know because I have never gotten a crash report before)

**Comment by Gegy** [comment]
Take a look at #158, a solution is provided there. :)
